freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_install.sh

```diff
@@ -26,8 +21,8 @@ ipa-server-install \
           --allow-zone-overlap \
           --ssh-trust-dns \
           --mkhomedir \
-          --ip-address "$IPADDR" \
-          $FORWARDERS \
+          --ip-address $IPADDR \
+          --auto-forwarders \
```

Comment:
@jamisonbennett correct, FreeIPA HA is not in tech-preview scope, even it is not in GA scope. Probably will file a Jira for GCP FreeIPA HA. My understanding is that FreeIPA HA is still under entitlement CDP_FREEIPA_HA.

Reply:
https://jira.cloudera.com/browse/CB-8677

Reply:
The problem is this is going into common code in the master branch and it will break FreeIPA HA for Azure & AWS.

Reply:
Can you put this in a platform if block in the template?

Reply:
My original comment says the forwarder change was not necessary on the primary but was just done for consistency. So this will not break anything.

Reply:
Had an offline chat. This does not break FreeIPA HA for Azure & AWS. The original description (of Jamison) says the forwarder change was not necessary on the primary but was just done for consistency. Any followups on this can be done as part of CB-8677